source/extensions/tracers/datadog/demo/http.js

```diff
@@ -0,0 +1,32 @@
+// This is an HTTP server that listens on port 8080 and responds to all
+// requests with some text, including the request headers as JSON.
+
+const http = require('http');
+const process = require('process');
+
+// In order for the span(s) associated with an HTTP request to be considered
+// finished, the body of the response corresponding to the request must have
+// ended.
+const ignoreRequestBody = request => {
+  request.on('data', () => {});
+  request.on('end', () => {});
+}
+
+const requestListener = function (request, response) {
```

Comment:
nit: any particular reason you are sometimes using `function` in this file and sometimes using `=>` ?

Reply:
I'm inconsistent, especially when the code began in another of my projects and then got extended to purpose here.

There are a few real differences between arrow functions and `function` in javascript, but mostly I use arrow functions for the short ones and `function` for the longer ones. Except when I'm inconsistent.